develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Add Pallet to the Runtime
+description: Learn how to include and configure pallets in a Polkadot SDK-based runtime, from adding dependencies to implementing necessary traits.
+---
+
+# Add Pallet to the Runtime
+
+## Introduction
+
+As demonstrated in the [Build a Local Solochain](/tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain/){target=\_blank} guide, the [Polkadot SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} provides a functional runtime that includes default FRAME development modules (pallets) to help you get started with building a custom blockchain.
```

Comment:
Not sure if we want to mention this. This section should go straight to the point and not depend on any template or previous section. It should be as generic as possible